test/simple.js

```diff
@@ -0,0 +1,13 @@
+const DeltaChat = require('../')
+const test = require('tape')
+const tempy = require('tempy')
+
+test('simple open test', t => {
+  const dc = new DeltaChat()
+  dc.open(tempy.directory(), err => {
```

Comment:
Can't run `docker` on osx for travis so create a simplified test so we know the binding works.